test/uk/gov/hmrc/plasticpackagingtaxregistrationfrontend/controllers/HelloWorldControllerSpec.scala

```diff
@@ -23,7 +39,8 @@ class HelloWorldControllerSpec extends AnyWordSpec with Matchers with GuiceOneAp
 
   val hello_world: hello_world = app.injector.instanceOf[hello_world]
 
-  private val controller = new HelloWorldController(appConfig, stubMessagesControllerComponents(), hello_world)
+  private val controller =
```

Comment:
I'd suggest extracting potentially common parts to separate objects as soon as possible. So when new people join your team you can show them "the path" to follow. You don't need to change it having a lot of dependent code.